packages/compass-collection-stats/src/components/index-stats-item/index-stats-item.jsx

```diff
@@ -45,9 +45,9 @@ class IndexStatsItem extends Component {
   render() {
     return (
       <div className={styles[LIST_CLASS]}>
-        <CollectionStatsItem label={INDEXES} value={this.props.indexCount} primary />
-        <CollectionStatsItem label={TOTAL_SIZE} value={this.props.totalIndexSize} />
-        <CollectionStatsItem label={AVG_SIZE} value={this.props.avgIndexSize} />
+        <CollectionStatsItem dataTestId="index-count" label={INDEXES} value={this.props.indexCount} primary />
```

Comment:
I noticed that other components that pass a dataTestId down to another component followed the camelCase convention so I copied it here. Then data-test-id="" would only be used with the html tag at the end.

Reply:
Personally I'd prefer us to switch to `data-test-id` everywhere to be consistent with html version, but if it's not a straightforward find/replace, then it's fine as-is

Reply:
Let me try..

Reply:
I did the find/replace now.

Reply:
nope nope nope. tests fail all over the place with enzyme finding the same thing twice. Revert ;)